release_tools/README.adoc

```diff
@@ -56,6 +56,20 @@ The last commit before the release has to have the `openscap-<version>` message
 This will create and push version tags, create new GitHub release and handle milestones swap.
 Finally, it will bump version numbers in `versions.sh` to be ready for the next upstream release.
 
+. Build OpenSCAP for Windows
++
+--
+The Windows build process is described in link:../docs/developer/developer.adoc[Developer Manual].
+Don't forget about setting the build type to Release using `-DCMAKE_BUILD_TYPE=Release`.
```

Comment:
@jan-cerny Since we merged a PR by @redhatrises that sets the `CMAKE_BUILD_TYPE=Release` if it is blank. Can you check if this is not needed and update the PR accordingly please? :) Thanks!

Reply:
It is not broken in the last AppVeyor build https://ci.appveyor.com/project/OpenSCAP/openscap/build/master-185
It might be only broken on my machine ...

Reply:
I am still unable to fix the build on my machine. I tried to investigate that, but I am not able to find anything in the CMake Cache or in the CPack log. @all Do you have any suggestions how to debug Windows Installer build? Any help would be appreciated.

Anyway, on AppVeyor CI, where it works OK, we explicitly set -DCMAKE_BUILD_TYPE=Release.
We could try to open a PR that would remove -DCMAKE_BUILD_TYPE=Release from `appveyor.yml` and we would see if it breaks AppVeyor. Based on that, we could decide if this needs to be mentioned in the docs or not.

Reply:
It looks like removing -DCMAKE_BUILD_TYPE=Release in PR https://github.com/OpenSCAP/openscap/pull/1127 haven't broken Windows Installer build in AppVeyor.

Reply:
So I guess I can remove it here.